src/features/defi/components/Deposit/Deposit.tsx

```diff
@@ -191,7 +191,7 @@ export const Deposit = ({
     [onContinue],
   )
 
-  const cryptoYield = calculateYearlyYield(apy, values.cryptoAmount)
+  const cryptoYield = calculateYearlyYield(bnOrZero(apy).toString(), values.cryptoAmount)
```

Comment:
suggestion: `calculateYearlyYield` already does a `bnOrZero()` call on `apy`, can update `apy` arg to `apy: string | undefined` instead?

Reply:
Changed to in [6b759a9](https://github.com/shapeshift/web/pull/7440/commits/6b759a95e9c9dfc87aedcf9f26d821336ca28f44)

```
function calculateYearlyYield(apy: string | undefined, amount: string = '') {
  return bnOrZero(amount)
    .times(apy ?? 0)
    .toString()
}
```